pallets/phala/src/stakepool.rs

```diff
@@ -1133,12 +1142,23 @@ pub mod pallet {
 				let to_distribute = rewards - commission;
 				if is_nondust_balance(to_distribute) {
 					pool_info.distribute_reward(to_distribute);
+					Self::deposit_event(Event::<T>::RewardReceived {
+						pid: pool_info.pid,
+						to_owner: commission,
+						to_stakers: to_distribute,
+					});
+					return;
 				} else if to_distribute > Zero::zero() {
 					Self::deposit_event(Event::<T>::RewardDismissedDust {
 						pid: pool_info.pid,
 						amount: to_distribute,
 					});
 				}
+				Self::deposit_event(Event::<T>::RewardReceived {
+					pid: pool_info.pid,
+					to_owner: commission,
+					to_stakers: Zero::zero(),
+				});
```

Comment:
Why will it go here? If the reward is dismissed (because it's dust), there's no reward distributed to any staker